src/CustomerioTracking.tsx

```diff
@@ -51,6 +51,14 @@ class CustomerIO {
       packageConfig.version = expoVersion;
     }
 
+    if (env.organizationId && env.organizationId != '') {
```

Comment:
We could do this,
```
!config.enableInApp && config.organizationId.isNotEmpty
```

Reply:
As discussed, we are keeping this just to add extra warning so customers remove `organizationId` when enabling using `enableInApp` flag.